internal/fs/vss_windows.go

```diff
@@ -810,6 +828,24 @@ func initializeVssCOMInterface() (*ole.IUnknown, error) {
 		}
 	}
 
+	// initialize COM security for VSS, this can't be called more then once
+	var capabilities uint32
+	if isPreWin8 {
+		capabilities = 0x00 // EOAC_NONE
+	} else {
+		capabilities = 0x20 // EOAC_STATIC_CLOAKING
+	}
+
+	if err = ole.CoInitializeSecurity(
+		-1,   // Allow *all* VSS writers to communicate back!
```

Comment:
Wrong comment. `CoInitializeSecurity` always sets a NULL security descriptor, which allows communication from all clients. This parameter is `cAuthSvc`. -1 here means that COM gets to decide which authentication services will be registered (https://learn.microsoft.com/en-us/windows/win32/api/combaseapi/nf-combaseapi-coinitializesecurity)